blivet/__init__.py

```diff
@@ -2575,6 +2576,9 @@ def turnOnSwap(self, rootPath=""):
                 try:
                     device.setup()
                     device.format.setup()
+                except UnknownSwapError as e:
+                    log.warn("Failed to activate swap on %s, skipping it.", device.path)
+                    break
```

Comment:
Wow, this is one of the weirdest ``while`` cycles I've ever seen. But that's not a problem of this change. :)